utils/abi/libstdc++/unwind-cxx.h

```diff
@@ -0,0 +1,401 @@
+// Extracted from the GCC's source code.
+
+// -*- C++ -*- Exception handling and frame unwind runtime interface routines.
+// Copyright (C) 2001-2022 Free Software Foundation, Inc.
+//
+// This file is part of GCC.
+//
+// GCC is free software; you can redistribute it and/or modify
```

Comment:
We can't incorporate this due to license conflict (the licenses are compatible, but the original contributor didn't sign our copyright license agreement). Maybe you can pick off the original noted below.

Reply:
I wrote a new header from scratch with necessary utilities after reading the Itanium ABI documentation.